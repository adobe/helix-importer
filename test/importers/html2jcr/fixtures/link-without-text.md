+-------------------------------------------------------------------------+
| link-without-text                                                       |
+=========================================================================+
| [Adobe](https://www.adobe.com)                                          |
+-------------------------------------------------------------------------+

+-------------------------------------------------------------------------+
| link-without-text                                                       |
+=========================================================================+
| [](https://www.adobe.com)                                               |
+-------------------------------------------------------------------------+
