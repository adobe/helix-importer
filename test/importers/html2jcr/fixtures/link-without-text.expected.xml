<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="link-without-text" link="https://www.adobe.com" model="link-without-text"/>
                <block_0 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="link-without-text" link="https://www.adobe.com" model="link-without-text"/>
            </section>
        </root>
    </jcr:content>
</jcr:root>